jupyter_releaser/lib.py

```diff
@@ -597,9 +597,9 @@ def prep_git(ref, branch, repo, auth, username, url):
         util.run(f"git switch -c {branch}")
 
     # Install the package
-    # install python package in editable mode with test deps
-    if util.SETUP_PY.exists():
-        util.run('pip install -q -e ".[test]"')
+    # install python package in editable mode with dev and test deps
+    if util.PYPROJECT.exists():
```

Comment:
Ah this seems to be causing issues with projects using a top-level `pyproject.toml` to manage configuration for Python monorepos (isort, tbump), without the top-level being installable.

Noticed in https://github.com/jupyterlite/jupyterlite/pull/644#issuecomment-1130508698

Reply:
Maybe there could be a new releaser configurable option to skip this installation step.

Reply:
Or the installation step moved to another step, so it can be `skip-`ed

Reply:
Hoisted to https://github.com/jupyter-server/jupyter_releaser/issues/321